addons/stock/views/stock_move_views.xml

```diff
@@ -337,8 +292,8 @@
                                     <field name="product_uom" options="{'no_open': True, 'no_create': True}" groups="uom.group_uom"/>
                                 </div>
                                 <field name="name" invisible="1"/>
-                                <field name="date_expected"/>
-                                <field name="date" attrs="{'invisible': [('state', '!=', 'done')]}"/>
+                                <field name="date" invisible="1"/>
```

Comment:
I would let the date visible (but readonly) at least to help resolving issue

Reply:
Yes, why not ? I will add that.